index.src.html

```diff
@@ -420,8 +429,11 @@ <h4 id="examples-post-signin">Post-sign-in Confirmation</h4>
               // the user agent to ask the user to store the credential for future use:
               var init = { method: "POST", credentials: c };
               fetch(e.target.action, init).then(r =&gt; {
-                if (/* |r| is a "successful" <a idl>Response</a> */)
+                if (/* |r| is a <a idl>Response</a> indicating a successful login. */) {
                   <a idl lt="store()">navigator.credentials.store</a>(c);
+                } else if (/* |r| indicates an expired credential */ {
```

Comment:
`s/expired/unsuccessful/` ? I expect you wouldn't want folks to call this only for expirations...